gh-80675: Set `f_trace_lines = True` on all frames upon `pdb.set_trace()` (#110881)
python/cpython: Misc/NEWS.d/next/Library/2023-10-14-20-15-53.gh-issue-80675._M-cQC.rst

> Set ``f_trace_lines = True`` on all frames upon :func:`pdb.set_trace()`

Reviewer: ```suggestion
Set ``f_trace_lines = True`` on all frames while there are active breakpoints.
```

does this make sense?

Author: That's not the case. We should always set `f_trace_lines = True` when `pdb.set_trace()` is used, or we won't be able to trigger a stop (regardless of breakpoints).
